src/Components/GraphContainer.jsx

```diff
@@ -23,9 +25,23 @@ const GraphContainer = ({ timeBounds, zoom }) => {
 	}, []);
 	const existsBadReading = !loading && (stats.length == 0 || Object.values(stats.at(-1).stats).includes(NaN));
 	//
+	let lastRetrieved;
+	const lastTime = stats.at(-1)?.unixTime;
+	if (typeof lastTime === 'undefined') {
+	lastRetrieved = "Data not retrieved."
+	}
+	else {
+		lastRetrieved = "Last Retrieved: " + new Date(lastTime * 1000).toLocaleString();
+	}
+
+// console.log(Date(stats[0].unixTime.toLocaleString()));
 
 	return (
 		<>
+		<Typography sx={{ my: 1 }} variant="h3" color="common.black">
+		{lastRetrieved}
```

Comment:
Using `h3` as the style makes it look a little too fat and blocky. How about `body1` or `body2`? Also, you shouldn't explicitly specify that it's a black color.